testing/io/test_terminalwriter.py

```diff
@@ -213,6 +213,12 @@ def test_should_not_do_markup_NO_COLOR_and_FORCE_COLOR(
     assert_color_not_set()
 
 
+def test_empty_NO_COLOR_ignored(monkeypatch: MonkeyPatch) -> None:
+    monkeypatch.setitem(os.environ, "NO_COLOR", "")
+    monkeypatch.setitem(os.environ, "FORCE_COLOR", "1")
```

Comment:
When setting both,the test doesn't actually cover all cases plus off hand I don't recall if our checks consider externally set variables

Reply:
I'm sorry but I don't understand the second part of your sentence. Could you rephrase, please?